docs/en/ingest-management/release-notes/release-notes-8.1.asciidoc

```diff
@@ -0,0 +1,174 @@
+// Use these for links to issue and pulls. 
+:kib-issue: https://github.com/elastic/kibana/issues/
+:kib-pull: https://github.com/elastic/kibana/pull/
+:agent-issue: https://github.com/elastic/beats/issues/
+:agent-pull: https://github.com/elastic/beats/pull/
+:fleet-server-issue: https://github.com/elastic/beats/issues/fleet-server/
+:fleet-server-pull: https://github.com/elastic/beats/pull/fleet-server/
+
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.1.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.1.0 relnotes
+
+[[release-notes-8.1.0]]
+== {fleet} and {agent} 8.1.0
+
+Review important information about the {fleet} and {agent} 8.1.0 release.
+
+[discrete]
+[[new-features-8.1.0]]
+=== New features
+
+The 8.1.0 release adds the following new and notable features.
+
+{agent}::
+* Add support for loading input configurations from external configuration files
+in standalone mode. You can load inputs from YAML configuration files under the
+folder `{path.config}/inputs.d`. {agent-pull}30087[#30087]
+
+[discrete]
+[[enhancements-8.1.0]]
+=== Enhancements
+
+{fleet}::
+* Add shipper label. {kib-pull}122491[#122491]
+* Add support for non-superuser access to *Fleet* and *Integrations*. {kib-pull}122347[#122347]
+* Add support for bundling packages as zip archives with {kib} source. {kib-pull}122297[#122297]
```

Comment:
I think we ought to remove this one due to adding this patch: https://github.com/elastic/kibana/pull/126699. This means that we don't actually invoke the bundled packages functionality yet, so this release note could be misleading for users. WDYT @kpollich @joshdover?

Reply:
This info is in the Kibana docs, too, which are already published. I think I will go ahead and merge this PR, and I can remove this item in a follow-up PR when I hear back. That way, I can remove it from Kibana, too. Just ping me if you want it removed. Thanks!

Reply:
@jen-huang I think this note can remain because all the code that supports installation from bundled packages still shipped here, even if we had to patch the actual bundled package assets out due to bugs. In 8.2 we can add a release note that we're bundling APM, Elastic Agent, Endpoint, Fleet Server, and Synthetics integration assets with the Kibana distributable.